mack/__init__.py

```diff
@@ -651,7 +651,7 @@ def constraint_append(
     if not isinstance(append_df, DataFrame):
         raise TypeError("You must provide a DataFrame that is to be appended.")
 
-    if not isinstance(quarantine_table, DeltaTable):
+    if not isinstance(quarantine_table, DeltaTable) and quarantine_table is not None:
```

Comment:
Shouldn't `is not None` be before `not isinstance`?

Reply:
fixed thanks.